test/e2e/stats_test.go

```diff
@@ -25,10 +25,6 @@ var _ = Describe("Podman stats", func() {
 		session := podmanTest.Podman([]string{"stats", "--no-stream", "123"})
 		session.WaitWithDefaultTimeout()
 		expect := `unable to get list of containers: unable to look up container 123: no container with name or ID "123" found: no such container`
```

Comment:
As long as you have to repush, though, might as well move the string below into `ExitWithError()`, and get rid of the temporary variable...